CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Version 0.4.1-dev (changes since 0.4.1 go here)
 
 ### Features
+* Added `skbio.tree.TreeNode.bifurcate` for converting multifurcating trees into bifurcating trees.
```

Comment:
Sorry.  Added

On Wed, Feb 17, 2016 at 10:09 AM, Jai Ram Rideout <notifications@github.com>
wrote:

> In CHANGELOG.md
> <https://github.com/biocore/scikit-bio/pull/904#discussion_r53205312>:
>
> > @@ -3,6 +3,7 @@
> >  ## Version 0.4.1-dev (changes since 0.4.1 go here)
> >
> >  ### Features
> > +* Added `skbio.tree.TreeNode.bifurcate` for converting multifurcating trees into bifurcating trees.
>
> issue number please
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio/pull/904/files#r53205312>.
>